TwitchPlaysAssembly/Src/UI/ModuleCameras.cs

```diff
@@ -326,6 +329,17 @@ private void Start()
 		TimerPrefab.color = modeColor;
 		timer.text.color = modeColor;
 		timer.StrikeIndicator.RedColour = modeColor;
+
+		if (TwitchPlaySettings.data.DarkMode) {
+			HeaderBackground.color = new Color32(0x0E, 0x0E, 0x10, 0xFF);
+			HeaderText.color = new Color32(0xEF, 0xEF, 0xEC, 0xFF);
+			MessagesBackground.color = new Color32(0x3A, 0x3A, 0x3D, 0xFF);
+		}
+		else {
+			HeaderBackground.color = new Color32(0xEE, 0xEE, 0xEE, 0xFF);
+			HeaderText.color = new Color32(0x4F, 0x4F, 0x4F, 0xFF);
+			MessagesBackground.color = new Color32(0xD8, 0xD8, 0xD8, 0xFF);
+		}
```

Comment:
Dark mode doesn't get applied until I go into a bomb which is a bit weird. It should probably apply as soon as possible. Also braces should be on a separate line for consistency.

Reply:
When exactly does the pending commands tab start?

Reply:
The UI for TP shows up as soon as TP is loaded so you should probably handle dark mode in the twitch service.

Reply:
I think it works now